scripts/us_bjs/nps/nps_statvars.mcf

```diff
@@ -0,0 +1,979 @@
+typeOf: dcs:StatisticalVariable
+institutionalization: dcs:Incarcerated
+statType: dcs:measuredValue
+measuredProperty: dcs:Count
+
+Node: dcid:Count_Jurisdiction_TwoOrMoreRaces_Female
+race: dcs:TwoOrMoreRaces
+gender: schema:Female
+populationType: dcs:Person
+measurementQualifier: dcs:MeasuredBasedOnJurisdiction
+typeOf: dcs:StatisticalVariable
+institutionalization: dcs:Incarcerated
+statType: dcs:measuredValue
+measuredProperty: dcs:Count
+
+Node: dcid:Count_Jurisdiction_Death_Female
+gender: schema:Female
+populationType: dcs:MortalityEvent
+measurementQualifier: dcs:MeasuredBasedOnJurisdiction
+typeOf: dcs:StatisticalVariable
+institutionalization: dcs:Incarcerated
+statType: dcs:measuredValue
+measuredProperty: dcs:Count
+
+Node: dcid:Count_Jurisdiction_DeathByJudicialExecution_Female
+causeOfDeath: dcs:JudicialExecution
+gender: schema:Female
+populationType: dcs:MortalityEvent
+measurementQualifier: dcs:MeasuredBasedOnJurisdiction
+typeOf: dcs:StatisticalVariable
+institutionalization: dcs:Incarcerated
+statType: dcs:measuredValue
+measuredProperty: dcs:Count
+
+Node: dcid:Count_Jurisdiction_DeathByIllnessOrNaturalCause_Female
+causeOfDeath: dcs:IllnessOrNaturalCause
+gender: schema:Female
+populationType: dcs:MortalityEvent
+measurementQualifier: dcs:MeasuredBasedOnJurisdiction
+typeOf: dcs:StatisticalVariable
+institutionalization: dcs:Incarcerated
+statType: dcs:measuredValue
+measuredProperty: dcs:Count
+
+Node: dcid:Count_Jurisdiction_DeathByAIDS_Female
+causeOfDeath: dcs:AIDS
+gender: schema:Female
+populationType: dcs:MortalityEvent
+measurementQualifier: dcs:MeasuredBasedOnJurisdiction
+typeOf: dcs:StatisticalVariable
+institutionalization: dcs:Incarcerated
+statType: dcs:measuredValue
+measuredProperty: dcs:Count
+
+Node: dcid:Count_Jurisdiction_DeathByIntentionalSelf-Harm(Suicide)_Female
+causeOfDeath: dcs:IntentionalSelf-Harm(Suicide)
+gender: schema:Female
+populationType: dcs:MortalityEvent
+measurementQualifier: dcs:MeasuredBasedOnJurisdiction
+typeOf: dcs:StatisticalVariable
+institutionalization: dcs:Incarcerated
+statType: dcs:measuredValue
+measuredProperty: dcs:Count
+
+Node: dcid:Count_Jurisdiction_DeathByAccidents(UnintentionalInjuries)_Female
+causeOfDeath: dcs:Accidents(UnintentionalInjuries)
+gender: schema:Female
+populationType: dcs:MortalityEvent
+measurementQualifier: dcs:MeasuredBasedOnJurisdiction
+typeOf: dcs:StatisticalVariable
+institutionalization: dcs:Incarcerated
+statType: dcs:measuredValue
+measuredProperty: dcs:Count
+
+Node: dcid:Count_Jurisdiction_DeathByDeathDueToAnotherPerson_Female
+causeOfDeath: dcs:DeathDueToAnotherPerson
+gender: schema:Female
+populationType: dcs:MortalityEvent
+measurementQualifier: dcs:MeasuredBasedOnJurisdiction
+typeOf: dcs:StatisticalVariable
+institutionalization: dcs:Incarcerated
+statType: dcs:measuredValue
+measuredProperty: dcs:Count
+
+Node: dcid:Count_Jurisdiction_DeathByAssault(Homicide)_Female
+causeOfDeath: dcs:Assault(Homicide)
```

Comment:
I think we should just do Homicide in this case, and we can encode that Homicide is a specializationOf/subClassOf/etc of Assault. I'll actually leave these comments for your schema PR (275)

Reply:
I didn't make this new schema entity, it was something that already existed

Reply:
ah ok, ty!